fb: Declare wfbFinishScreenInit, wfbScreenInit for !FB_ACCESS_WRAPPER

xorg-x11-drv-nouveau wfbScreenInit without defining FB_ACCESS_WRAPPER
(which has other unintended side effects).  Presently, this compiles
and links because compilers still support implicit function
declarations, but this is going to change fairly soon.  This seems to
be the most straightforward change to keep the driver building.

Submitted upstream:

  <https://gitlab.freedesktop.org/xorg/xserver/-/merge_requests/1114>

diff -ur xorg-server-1.20.14.orig/fb/fb.h xorg-server-1.20.14/fb/fb.h
--- xorg-server-1.20.14.orig/fb/fb.h	2021-12-15 20:01:24.000000000 +0100
+++ xorg-server-1.20.14/fb/fb.h	2023-04-13 13:59:47.325341537 +0200
@@ -1027,7 +1027,6 @@
                int dpiy, int width,     /* pixel width of frame buffer */
                int bpp);        /* bits per pixel of frame buffer */
 
-#ifdef FB_ACCESS_WRAPPER
 extern _X_EXPORT Bool
 wfbFinishScreenInit(ScreenPtr pScreen,
                     void *pbits,
@@ -1049,7 +1048,6 @@
               int width,
               int bpp,
               SetupWrapProcPtr setupWrap, FinishWrapProcPtr finishWrap);
-#endif
 
 extern _X_EXPORT Bool
 fbFinishScreenInit(ScreenPtr pScreen,
